src/lib/components/SvgSnakeBody.svelte

```diff
@@ -26,153 +33,218 @@
       body.pop();
     }
 
+    // Split wrapped body parts into separated segments
+    const bodySegments = ((body: Point[]): Point[][] => {
```

Comment:
Might be nice to extract nicely named function for this. Maybe `splitBodySegments`?

I didn't see at first that this was a self-evaluating function

Reply:
Applied